src/java_bytecode/java_bytecode_convert_method.h

```diff
@@ -53,6 +53,7 @@ void java_bytecode_convert_method_lazy(
   const symbolt &class_symbol,
   const irep_idt &method_identifier,
   const java_bytecode_parse_treet::methodt &,
-  symbol_tablet &symbol_table);
+  symbol_tablet &symbol_table,
+  message_handlert &);
```

Comment:
Nit: Even though it's not strictly necessary, I tend to prefer naming the parameters in the declaration - e.g. `message_handlert &message_handler` - but I note that the `java_bytecode_parse_treet::methodt` parameter hasn't been named either (I wouldn't be against you doing a drive-by cleanup of that at the same time ;-) )

Reply:
I asked about the general guidelines for this recently and I was told that the names can be left out unless it's not clear from the type what the parameter represents. For example, from `const irep_idt &` it would not be clear that the parameter is the method identifier, thus the name is given. But for `const java_bytecode_parse_treet::methodt &` it's pretty clear what it is. Since the message handler does not have any 'hidden' role here, I would probably leave it.

Reply:
Fine by me then :-)